README.md

```diff
@@ -1,4 +1,5 @@
 # NNtrainer
+[![Code Coverage](http://ec2-54-180-96-14.ap-northeast-2.compute.amazonaws.com/nntrainer/ci/badge/codecoverage.svg)](http://ec2-54-180-96-14.ap-northeast-2.compute.amazonaws.com/nntrainer/ci/gcov_html/index.html)
```

Comment:
BTW, it seems that `http://nnsuite.mooo.com/nntrainer` is not working. Is it different from `http://nnsuite.mooo.com/nnstreamer`?

Reply:
@dongju-chae NNT uses the aws-dev server not the aws server, nnsuite.mooo.com.

Reply:
Got it

Reply:
BTW, it would be great if we have an additional URL alias such as `nnsuitedev.mooo.com`